### DISCUSSION TITLE
Is there a way to not cached on some specific endpoints

### DISCUSSION THREAD

**Discussion by sinkaroid** [requests-cache/aiohttp-client-cache#139] [Q&A] [answered]
I have a bunch of api endpoints, some of them is `/random` and `?shuffle` which is cached and the random flow is not going well because the object still same in cache  

How to validate them with only `SQLiteBackend` flow, I dont want to make optional statements like `if abc_some_endpoint == "/random"`

**Comment by sinkaroid** [comment]
![Screenshot_89](https://user-images.githubusercontent.com/12372481/208251583-b9fbd315-49a4-4640-ab3e-ce6654e193a5.png)

Looks like solved with this, is that mean if i fill with 0 is 0 minutes? what is -1 mean

**Reply by JWCook** [reply, marked as answer]
Yes, currently the best option is to use URL patterns to set an expiration of 0. for example:
```python
cache = SQLiteBackend(
    urls_expire_after={
        '/random': 0,
        '*': 360,   # Default expiration for all other URLs
    }
)
```

That will likely do what you want, but it's not 100% the same as disabling the cache for that `/random` endpoint. The response will be stored in the cache, but it won't actually be used unless you specifically request it. The reason for that is to support things like `max-stale` or [conditional requests](https://developer.mozilla.org/en-US/docs/Web/HTTP/Conditional_requests) (not yet implemented in this library, but mentioned in #79 if anyone needs it). 

> what is -1 mean

That means "never expires." Docs for the different expiration values are here:  https://aiohttp-client-cache.readthedocs.io/en/stable/user_guide.html#cache-expiration

Hope that helps.
